pkg/kubelet/pleg/evented.go

```diff
@@ -35,7 +35,9 @@ import (
 // The frequency with which global timestamp of the cache is to
 // is to be updated periodically. If pod workers get stuck at cache.GetNewerThan
 // call, after this period it will be unblocked.
-const globalCacheUpdatePeriod = 5 * time.Second
+// This is the same values as the relist period of Generic PLEG
+// in order to unblock pod workers as frequent as Generic PLEG.
+const globalCacheUpdatePeriod = 1 * time.Second
```

Comment:
If Evented PLEG also relists every 1 second then it is as good as just having Generic PLEG right? https://github.com/kubernetes/kubernetes/blob/a499facee693a1a83daadb82d88f7b51d324ffc5/pkg/kubelet/kubelet.go#L193 

What is the advantage of using Evented PLEG in that case?